package schneider.davi.to_do_app.exception;

public record DefaultErrorMessage(int status, String message) {
}
